polaris-react/src/components/Tabs/components/Item/Item.tsx

```diff
@@ -24,12 +24,12 @@ export const Item = memo(function Item({
   const focusedNode = useRef<HTMLButtonElement | ReactElement | null>(null);
 
   useEffect(() => {
-    if (
-      focusedNode.current &&
-      focusedNode.current instanceof HTMLElement &&
-      focused
-    ) {
-      focusedNode.current.focus();
+    const focusTarget = focusedNode.current;
+
+    if (focusTarget && focusTarget instanceof HTMLElement && focused) {
+      requestAnimationFrame(() => {
```

Comment:
[Seems to be the same issue as popover overlay](https://github.com/Shopify/polaris/blob/main/polaris-react/src/components/Popover/components/PopoverOverlay/PopoverOverlay.tsx#L181)